packages/@lwc/ssr-compiler/src/compile-js/remove-decorator-import.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import { ImportDeclaration } from 'estree';
+import { NodePath, builders as b } from 'estree-toolkit';
+
+export function removeDecoratorImport(path: NodePath<ImportDeclaration>) {
+    if (!path.node || path.node.source.value !== '@lwc/ssr-runtime') {
+        return;
+    }
+
+    const filteredSpecifiers = path.node.specifiers.filter(
+        (specifier) =>
+            !(
+                specifier.type === 'ImportSpecifier' &&
+                specifier.imported.type === 'Identifier' &&
+                (specifier.imported.name === 'api' || specifier.imported.name === 'wire')
```

Comment:
```suggestion
                (['api', 'wire', 'track'].includes(specifier.imported.name))
```

We missed `track`.

Reply:
`track` can't be universally removed because it can be used as a non-decorator.

https://github.com/salesforce/lwc/blob/82467e330a85036c1f4253473754e5b634e92bce/packages/%40lwc/engine-core/src/framework/decorators/track.ts#L18-L25

https://github.com/salesforce/lwc/blob/82467e330a85036c1f4253473754e5b634e92bce/packages/%40lwc/integration-types/src/decorators/track.ts#L10-L11

Reply:
```suggestion
                // Excluding track because it also has a non-decorator signature that might be used
                (specifier.imported.name === 'api' || specifier.imported.name === 'wire')
```